README.rst

```diff
@@ -100,6 +106,18 @@ Setup (integrate in an existing django project)
 
     urlpatterns += staticfiles_urlpatterns()
 
+Testing (locally)
```

Comment:
Testing can be confused with how to run automated tests, could you replicate the ["Installing for development" section of django-netjsonconfig](https://github.com/openwisp/django-netjsonconfig/#installing-for-development)?